src/Workspaces/Core/Desktop/project.json

```diff
@@ -2,7 +2,9 @@
   "dependencies": {
     "ManagedEsent": "1.9.4",
     "Microsoft.Build": "15.1.0-preview-000458-02",
-    "Microsoft.Build.Tasks.Core": "15.1.0-preview-000458-02"
+    "Microsoft.Build.Tasks.Core": "15.1.0-preview-000458-02",
+    "System.Data.Common": "4.3.0",
+    "System.Data.SQLite": "1.0.104"
```

Comment:
For code running in your ServiceHub process, you can pick any version of sqlite you want since your process is isolated. But if you're loading this in VS, yes, you should sync up with our team and P&T to make sure it's the same version we already ship with VS.
FWIW, you might want to use the same version VS already uses within your service anyway, in case it can leverage reusing memory for the shared image across processes (I think Windows can do that). In which case, I suggest you email Viktor Veis to ask what version they use.